src/widgets/_views.scss

```diff
@@ -42,6 +42,7 @@ scrolledwindow {
 .view {
     background: #{'@base_color'};
     color: $fg-color;
+    -gtk-secondary-caret-color: #{'@accent_color_100'};
```

Comment:
this should probably instead be:
```suggestion
    -gtk-secondary-caret-color: #{'@selected_bg_color'};
```

Reply:
Oh there is a dedicated variable already, thank you for your suggestion!